pkg/network/network_test.go

```diff
@@ -0,0 +1,66 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("Network", func() {
+	Describe("Internal Functions", func() {
```

Comment:
Once you move it to a test module dedicated to kubemacpool, don't forget to rename these. You can drop the inner Describe and rename outer